sympy/concrete/summations.py

```diff
@@ -459,6 +460,16 @@ def is_convergent(self):
             if p2_series_test[p] <= 1:
                 return S.false
 
+        ### ------------- Limit comparison test -----------###
+        # (1/n) comparison
+        try:
+            lim_comp = limit(sym*sequence_term, sym, S.Infinity)
+            if lim_comp.is_number:
+                if lim_comp > 0:
```

Comment:
This can be combined with the previous line: `if lim_comp.is_number and lim_comp > 0:`.

Reply:
Done @jksuom